application.py

```diff
@@ -1,42 +1,49 @@
+import logging
 import os
 from PyQt5.QtWidgets import QMainWindow, QVBoxLayout, QWidget, QLabel, QGraphicsOpacityEffect
-from PyQt5.QtCore import QTimer, Qt, QPropertyAnimation
+from PyQt5.QtCore import QTimer, Qt, QPropertyAnimation,  QTime
 from PyQt5.QtGui import QPixmap
 import qrcode
 import requests
 from services import Services
 from video_player import VideoPlayer
-import logging
-
-
-service = Services()
+from diagnostic_menu import DiagnosticMenu
+import json
 
 BASE_URL = 'https://api.olhar.media/'
 BASE_URL_VIDEO_ENDED = 'https://api.olhar.media/?regview=1'
 ASSETS_FOLDER = "./assets"
-CURRENT_CITY = service.get_city_from_coordinates(*service.get_lat_lon())
-
-
-logger = service.get_logger('application')
-logger.info(f'Current city: {CURRENT_CITY}')
-
 
 class App(QMainWindow):
+
     def __init__(self):
         super().__init__()
         self.setWindowTitle("Видеоплеер")
         self.showFullScreen()
+
+        self.service = Services()
+        self.current_city = self.service.get_current_city()
+
+        self.logger = logging.getLogger('application')
```

Comment:
I meant, you define the `logger` in the global scope once per file like you did before, but you call `logging.basicConfig` only once from wherever you need (usually as early as you can (which is right after you get logging configuration if it is not hardcoded (which means in the very beginning of the `main` function if it is all hardcoded)))